packages/api/tests/lambdas/test-sqs-message-consumer.js

```diff
@@ -82,6 +81,9 @@ async function cleanupRulesAndQueues(rules, queues) {
     rules.map((rule) => rulesModel.delete(rule))
   );
 
+  // Delete queueName for each object in list
+  queues.forEach((q) => delete q.queueName);
```

Comment:
Why is this necessary?

Reply:
This is necessary because `createSqsQueues` returns `queueName` and, with the code that follows, if not removed, will throw an error because it won't be able to find a queue of `queueName` to delete:
```
  const queueUrls = queues.reduce(
    (accumulator, currentValue) => accumulator.concat(Object.values(currentValue)), []
  );

  await Promise.all(
    queueUrls.map((queueUrl) => SQS.deleteQueue(queueUrl))
  );
``

Reply:
OK. I'm still not sure if the code is working even if you delete `q.queueName`. `deleteQueue` expects a queue URL as a single argument, whereas right now, I think it might be receiving a object where `queueUrl` is a property. 

Regardless I think we can get rid of the `forEach` and `reduce` by doing this:

```
  await Promise.all(
    queues.map((queue) => SQS.deleteQueue(queue.queueUrl))
  );
```

The `reduce` was just trying to prepare an array of just queue URLs, but that's totally unnecessary since we can read the object property for `queueUrl` and pass it in to `deleteQueue`

Reply:
`queueUrls` is creating an array of the `queueUrl`s and `deadLetterQueueUrl`s, so ` queueUrls.map((queueUrl) => SQS.deleteQueue(queueUrl))` is correctly deleting each queue with the current implementation

Reply:
I see. It's because we want to delete both the dead letter queue and regular queue. In that case I still think this code is unnecessarily confusing and would prefer something like:

```
  await Promise.all(
    queues.map(async (queue) => {
      await SQS.deleteQueue(queue.queueUrl);
      await SQS.deleteQueue(queue.deadLetterQueueUrl);
    })
  );
```

which is a lot easier to understand and removes the need for the `forEach` or `reduce`

Reply:
[`c9a4bfe` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/c9a4bfe0055910a944af43fff5bf4051bb5857c9)